### PR TITLE
Add metrics docs for Python SDK

### PR DESCRIPTION
**Pull request by iambriccardo**

### CODE REVIEW

File: src/platforms/python/common/metrics/index.mdx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+Sentry metrics help you pinpoint and solve issues that impact user experience and app performance by measuring the data points that are important to you. You can track things like processing time, event size, user signups, and conversion rates, then correlate them back to tracing data in order to get deeper insights and solve issues faster.
+
+## Emit a Counter
+
+Counters are one of the more basic types of metrics and can be used to count certain event occurrences.
+
+To emit a counter, do the following:
+
+```php
```
Comment (ale-cota) [inline src/platforms/python/common/metrics/index.mdx:20]: I assume this is a copy/pasta mistake.
```suggestion
```python
```

File: src/platforms/python/common/metrics/index.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+)
+```
+
+## Emit a Distribution
+
+Distributions help you get the most insights from your data by allowing you to obtain aggregations such as `p90`, `min`, `max`, and `avg`.
+
+To emit a distribution, do the following:
+
+```php
```
Comment (ale-cota) [inline src/platforms/python/common/metrics/index.mdx:38]:
```suggestion
```python
```

File: src/platforms/python/common/metrics/index.mdx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+)
+```
+
+## Emit a Set
+
+Sets are useful for looking at unique occurrences and counting the unique elements you added.
+
+To emit a set, do the following:
+
+```php
```
Comment (ale-cota) [inline src/platforms/python/common/metrics/index.mdx:56]:
```suggestion
```python
```

File: src/platforms/python/common/metrics/index.mdx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+)
+```
+
+## Emit a Gauge
+
+Gauges let you obtain aggregates like `min`, `max`, `avg`, `sum`, and `count`. They can be represented in a more space-efficient way than distributions, but they can't be used to get percentiles. If percentiles aren't important to you, we recommend using gauges.
+
+To emit a gauge, do the following:
+
+```php
```
Comment (ale-cota) [inline src/platforms/python/common/metrics/index.mdx:74]:
```suggestion
```python
```

File: src/platforms/python/common/metrics/index.mdx [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+)
+```
+
+## Emit a Timer
+
+Timers can be used to measure the execution time of a specific block of code. They're implemented like distributions, but measured in seconds.
+
+To emit a timer, do the following:
+
+```php
```
Comment (ale-cota) [inline src/platforms/python/common/metrics/index.mdx:92]:
```suggestion
```python
```

**Review by ale-cota — CHANGES_REQUESTED** [review]

File: src/platforms/python/common/metrics/index.mdx [hunk 6]
```diff
@@ -0,0 +1,152 @@
+---
+title: Set Up Metrics
+description: "Learn how to measure the data points you care about by configuring Metrics in your Python app."
+---
+
+<Note>
+
+Metrics for Python are supported with Sentry Python SDK version `1.38.0` and above.
```
Comment (cleptric) [inline src/platforms/python/common/metrics/index.mdx:8]: Let's put `1.40.0` here to simplify it. It gets confusing pointing out various versions on the same page.

**Pull request merged**